security/acme-client/src/opnsense/mvc/app/controllers/OPNsense/AcmeClient/forms/dialogValidation.xml

```diff
@@ -1065,6 +1065,10 @@
         <type>header</type>
         <style>table_dns table_dns_transip</style>
     </field>
+    <field>
+        <label>NOTE: A DNS sleep time of at least 300 is recommended.</label>
+        <type>header</type>
+        <style>table_dns table_dns_transip</style>
```

Comment:
I think this `<style>` tag can be omitted, because the previous tag already added it.